client/my-sites/stats/activity-log/index.jsx

```diff
@@ -438,6 +436,7 @@ class ActivityLog extends Component {
 							total={ logs.length }
 						/>
 						<section className="activity-log__wrapper">
+							{ siteIsOnFreePlan && <div className="activity-log__fader" /> }
```

Comment:
Are we fading real events here?

Reply:
yes we are!

Reply:
Why not just show them?

Reply:
We need to indicate there are more, but the amount shown is limited until they upgrade.